app/stac_api/serializers.py

```diff
@@ -802,10 +802,13 @@ class Meta:
     aborted = serializers.SerializerMethodField()
 
     def validate(self, attrs):
-        # TODO BGDIINF_SB-1983 md5_parts should be required not optional
+        # get partial from kwards (if partial true and no md5 : ok, if false no md5 : error)
```

Comment:
picky, sorry, just a small typo here. Guess it should be kwargs instead of kwards ;-)